vscode-extension/src/extension.ts

```diff
@@ -757,18 +757,33 @@ async function setupEnvironmentVariables(context: vscode.ExtensionContext) {
     return;
   }
 
+  const envTemplatePath = vscode.Uri.joinPath(
+    context.extensionUri,
+    "static",
+    "env_template.env"
+  );
+
   if (fs.existsSync(envPath)) {
-    vscode.window.showInformationMessage(
-      "Env file already exists, will implement next PR"
-    );
+    var helperText: string = "\ntest, will change next PR";
+    // fs.readFile(envTemplatePath.fsPath, function read(err, data) {
```

Comment:
nit: Remove commented out code

Reply:
I change it in the next PR, just wasn't able to get it to work and wanted quick feedback